plugins/experimental/slice/client.cc

```diff
@@ -74,7 +75,7 @@ handle_client_req(TSCont contp, TSEvent event, Data *const data)
         data->m_statustype = TS_HTTP_STATUS_REQUESTED_RANGE_NOT_SATISFIABLE;
 
         // First block will give Content-Length
-        rangebe = Range(0, data->m_config->m_blockbytes);
+        rangebe = Range(0, conf->m_blockbytes);
       }
     } else {
       DEBUG_LOG("%p Full content request", data);
```

Comment:
Please mitigate the reference to `SLICER_MIME_FIELD_INFO` on line 84 below.

Reply:
stupid missed this.